src/senselab/audio/tasks/speech_enhancement/api.py

```diff
@@ -10,27 +11,31 @@
 
 
 def enhance_audios(
-    audios: List[Audio], 
-    model: SenselabModel = SpeechBrainModel(
-        path_or_uri="speechbrain/sepformer-wham16k-enhancement", 
-        revision="main"), 
-    device: Optional[DeviceType] = None
+    audios: List[Audio],
+    model: SenselabModel = SpeechBrainModel(path_or_uri="speechbrain/sepformer-wham16k-enhancement", revision="main"),
+    enhancer: Optional[SpeechBrainEnhancer] = None,
```

Comment:
what is this?

Reply:
That's what I was referring to when I said:
"I've been thinking about this a bit, and I'm wondering if we could create a shared enhancer object that uses the same model using multiprocessing.Manager and update the API to accept it as a parameter. I went ahead and updated the API (easy to change it back, and it doesn't affect existing usage)."

Reply:
Should I change it back?

Reply:
I changed it back